src/Colors.stories.mdx

```diff
@@ -0,0 +1,49 @@
+<!-- Colors.stories.mdx -->
```

Comment:
You should:
- rename folder `scss` to `styles`
- put this story (and the next style stories) in the `styles` folder